logrotate.c

```diff
@@ -2195,6 +2195,7 @@ static int writeState(const char *stateFilename)
 	if (bytes < 0)
 		error = bytes;
 
+/* time in seconds it takes earth to go around sun */
```

Comment:
The commit message says that you are adding a comment to explain where ```31556926``` comes from but it does not.  I think it is obvious that it takes a year till Earth goes around the Sun.  The information missing here is how the number was actually computed (or which resource was used to pick a pre-computed number from).